src/components/modals/MintModal/index.tsx

```
@@ -0,0 +1,390 @@
+                <MultipleSelectNative
+                  options={accounts}
+                  onChange={handleChange}
+                />
+              </Grid>
+              <Grid item xs={12}>
+                <Typography variant="h6">Contract Details</Typography>
+              </Grid>
+              {accounts
+                .filter((el) => el.contractId === Number(contractId))
+                .map((el) => (
+                  <Grid item xs={12}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)